src/resources/resources.css

```diff
@@ -81,9 +82,77 @@
 	display: block;
 }
 
+/* TODO: Add pagination css */
 .pagination-container {
 	margin: auto;
+}
+
+/* List Buttons To Render Items from Database */
+.resource-list-item {
 	display: flex;
+	flex-direction: column;
+	padding-left: 0rem;
 }
 
-/* TODO: Add pagination css */
+.resource-list-title {
+	position: relative;
+	text-align: left;
+	display: block;
+	padding: 2rem 0.75rem 2rem 0.75rem;
```

Comment:
padding top and bottom 2rem is quite big. I recommend it should be within 0.75-1rem. If padding for all sides are 0.75rem, you just need to add `padding: 0.75rem`. See comment at line 72.

Reply:
Changed to:
padding: 1rem 0.75rem 1rem 0.75rem;